.github/workflows/testing_gsoc.yml

```diff
@@ -16,8 +16,6 @@ env:
 jobs:
   Test-MSS-GSOC:
     runs-on: ubuntu-latest
-    secrets:
```

Comment:
does this make problems?

Reply:
it is principal running, so it may be not needed. It was not cleaning that for me. So it may be ok.